homework/index.js

```diff
@@ -30,15 +32,138 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, repositories) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-        return;
-      }
-      createAndAppend('pre', root, { text: JSON.stringify(repositories, null, 2) });
+  function openNewTab(url) {
+    window.open(url, '_blank');
+  }
+
+  function bindContributions(contributorsUrl) {
+    const repoInfoContainer = document.getElementById('repo-info-container');
+    const contributorsDiv = createAndAppend('div', repoInfoContainer, {
+      class: 'contributors-info',
+    });
+    fetchJSON(contributorsUrl)
+      .then(response => {
+        const div = createAndAppend('div', contributorsDiv);
+        createAndAppend('h4', div, { text: 'Contributions' });
+        createAndAppend('hr', div);
+
+        const table = createAndAppend('ul', div, { class: 'grid' });
+        const contributors = response;
+        contributors.forEach((_contributor, index) => {
+          const imageUrl = contributors[index].avatar_url;
+          const name = contributors[index].login;
+          const contributionsCount = contributors[index].contributions;
+          const profileURL = contributors[index].html_url;
+          const tr = createAndAppend('li', table, { class: 'clickable row' });
```

Comment:
Again, the name `tr` is confusing because you are not creating a `<tr>`, you are creating an `<li>` instead.